src/components/index.ts

```diff
@@ -0,0 +1,16 @@
+import { CustomButton } from './CustomButton/CustomButton';
+import { Logo } from './Logo/Logo';
+import { CustomAppBar } from './CustomAppBar/CustomAppBar';
+import { ActiveNote } from './ActiveNote/ActiveNote';
+import { EditMenuContainer } from './EditMenu/EditMenuContainer';
+import { DisplayedNote } from './DisplayedNote/DisplayedNote';
+import { CreateNoteContainer } from './CreateNote/CreateNoteContainer';
```

Comment:
add index.ts files inside each folder so you can import them like this
` import { CustomButton } from './CustomButton';`